.gitignore

```diff
@@ -33,6 +33,7 @@ nbproject
 .sass-cache
 *.map
 _site
+_tmp_preview_config.yml
```

Comment:
I think this is to generate the S3 sandbox version, is that correct @andy-armstrong ?